### ISSUE TITLE
alter table order at the end & invalid SQL

### ISSUE THREAD

**Issue by dataf3l** [DBDiff/DBDiff#103]
output looks like this:

ALTER TABLE `tablename` ENGINE = ;
ALTER TABLE `tablename` ENGINE = ;
ALTER TABLE `tablename` ENGINE = ;

that doesn't look like valid SQL.

ALTER TABLE `table1` DROP `field1`;
ALTER TABLE `table2` DROP `id`;
ALTER TABLE `table1` DROP `field2`;

this makes hard to understand.

can it be changed so it looks like this?

ALTER TABLE `table1` DROP `field1`;
ALTER TABLE `table1` DROP `field2`;
ALTER TABLE `table2` DROP `id`;

in other words, sorted by table?

this is so that the statements are easier to grok by humans.
can anybody help.
